doc/metadata_routing.rst

```diff
@@ -9,42 +9,59 @@ Metadata Routing
 ================
 
 .. note::
-  The Metadata Routing API is experimental, and is not implemented yet for many
+  The Metadata Routing API is experimental, and is not implemented yet for some
   estimators. Please refer to the :ref:`list of supported and unsupported
   models <metadata_routing_models>` for more information. It may change without
   the usual deprecation cycle. By default this feature is not enabled. You can
-  enable this feature  by setting the ``enable_metadata_routing`` flag to
+  enable it by setting the ``enable_metadata_routing`` flag to
   ``True``::
 
     >>> import sklearn
     >>> sklearn.set_config(enable_metadata_routing=True)
 
-This guide demonstrates how metadata such as ``sample_weight`` can be routed
-and passed along to estimators, scorers, and CV splitters through
-meta-estimators such as :class:`~pipeline.Pipeline` and
-:class:`~model_selection.GridSearchCV`. In order to pass metadata to a method
-such as ``fit`` or ``score``, the object consuming the metadata, must *request*
-it. For estimators and splitters, this is done via ``set_*_request`` methods,
-e.g. ``set_fit_request(...)``, and for scorers this is done via the
-``set_score_request`` method. For grouped splitters such as
-:class:`~model_selection.GroupKFold`, a ``groups`` parameter is requested by
-default. This is best demonstrated by the following examples.
-
-If you are developing a scikit-learn compatible estimator or meta-estimator,
-you can check our related developer guide:
-:ref:`sphx_glr_auto_examples_miscellaneous_plot_metadata_routing.py`.
-
-.. note::
```

Comment:
why remove the `.. note::` directive?

Reply:
Because I merged that note with the other one on top. It's cleaner to read like that.

Reply:
Then you need to remove the indentation here.

Reply:
Hm, I cannot see why you suggest that.
I made those two notes into a larger one in the beginning of the document. All of that is supposed to be one note:

```
.. note::
  The Metadata Routing API is experimental, and is not implemented yet for some
  estimators. Please refer to the :ref:`list of supported and unsupported
  models <metadata_routing_models>` for more information. It may change without
  the usual deprecation cycle. By default this feature is not enabled. You can
  enable it by setting the ``enable_metadata_routing`` flag to
  ``True``::

    >>> import sklearn
    >>> sklearn.set_config(enable_metadata_routing=True)

  Note that the methods and requirements introduced in this document are only
  relevant if you want to pass :term:`metadata` (e.g. ``sample_weight``) to a method.
  If you're only passing ``X`` and ``y`` and no other parameter / metadata to
  methods such as :term:`fit`, :term:`transform`, etc., then you don't need to set
  anything.
```

If I remove the indentation from the second paragraph, then it's not included in the note anymore.